### DISCUSSION TITLE
How to I skip bundling during deploy?

### DISCUSSION THREAD

**Discussion by vicary** [floydspace/serverless-esbuild#213] [Q&A] [answered]
Since you support `sls package`, is there a way for me to grab the `.serverless` folder and skip ahead to the deploy stage in a later time?

**Comment by floydspace** [comment, marked as answer]
hi @vicary. 
If I catch it right. you can use `sls package` to prepare `.serverless` artifacts and then use `sls deploy --package .serverless` to deploy artifacts at a later time

**Reply by vicary** [reply]
That's exactly what I wanted, thanks a lot!
